build.sbt

```diff
@@ -128,6 +133,43 @@ def youngestForwardCompatible(subProj: String) = {
   }.toSet
 }
 
+/**
+ * Generic AnyVal's were incorrectly reported as the underlying type in
```

Comment:
@regadas these are the MiMa suppressions that work for the AnyVal bug from Java, if you want to do this independently first.

Reply:
Hi @martijnhoekstra sorry for the late reply; Yeah I think we should do the `2.12.10` bump separate and add these rules.

Reply:
Sure, I'll make a separate PR